tests/jballerina-unit-test/src/test/resources/test-src/statements/arrays/array_size_test.bal

```diff
@@ -21,4 +21,6 @@ function arraySizeTest() {
     int[2][constLength] f = [[1,2],[1,2]];
     int[constLength][2] k = [[1,2],[1,2]];
     int[2][constLength][2] p = [[[1,2],[1,2]],[[1,2],[1,2]]];
-}
\ No newline at end of file
+    int[0x2] q = [1,2];
```

Comment:
```suggestion
    int[0x2] q = [1, 2];
```

Shall we also add some tests assigning to an array of the exact size with the size specified as a decimal number. Just to assert correctly parsing.

Reply:
I'll add.